composer.json

```diff
@@ -11,7 +11,7 @@
         }
     ],
     "require": {
-        "php": ">=7.2.5",
+        "php": ">=8.0",
```

Comment:
I don't think we should drop PHP 7 compact. Any way to make the code work for all currently supported versions?

Reply:
I fixed it